server/mdm/apple/commander.go

```diff
@@ -161,6 +161,32 @@ func (svc *MDMAppleCommander) EraseDevice(ctx context.Context, host *fleet.Host,
 	return nil
 }
 
+func (svc *MDMAppleCommander) InstallApplication(ctx context.Context, hostUUIDs []string, uuid string, adamID string) error {
+	raw := fmt.Sprintf(`<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Command</key>
+    <dict>
+        <key>ManagementFlags</key>
+        <integer>0</integer>
```

Comment:
Is this the [docs](https://developer.apple.com/documentation/devicemanagement/installapplicationcommand/command) for this command? It's confusing to me since it says possible values for `ManagementFlags` are `1,4,5`.

Reply:
Not sure, @roperzh does that look like the right doc? 

fwiw testing this out, I didn't have any issues with the command not going through because of this.

Reply:
this value was provided by @marko-lisica , but I don't know if it's intentional to force the "default" value or something like that.


guess is that it doesn't hurt, but you won't get the behavior described in the docs either.

Reply:
yeah, that sounds right @roperzh . I'd say we should leave it, since we don't have the behaviors described in that doc spec'd anywhere. 

We can add a cleanup item to confirm this with Marko tomorrow.

Reply:
@roperzh @jahzielv @gillespi314 This command XML is from Apple's docs ([see here](https://developer.apple.com/documentation/devicemanagement/install_an_app#3383808)).

Reply:
I think we'll probably want to change this command in the future when we learn more. There are a couple of flags that we'll probably need to include/change: (`InstallAsManaged`, `ManagementFlags`, `ChangeManagementState`)